skins/cat17/src/scripts/donationForm.js

```diff
@@ -291,13 +286,11 @@ $( function () {
     if ( !addressIsValid() ) {
       if ( formContent.addressType === 'person' ) {
         store.dispatch( actions.newMarkEmptyFieldsInvalidAction(
-          [ 'salutation', 'firstName', 'lastName', 'street', 'postcode', 'city', 'email' ],
-          [ 'companyName' ]
```

Comment:
This is a dead argument or what?

Reply:
No - that's an oversight. Have to look at it again. Thanks